mr/src/test/resources/log4j2.properties

```diff
@@ -77,4 +77,9 @@ logger.Zookeeper.level=warn
 logger.Storm-Shaded.name=org.apache.storm.shade
 logger.Storm-Shaded.level=warn
 #logger.OpenSearch-Storm.name=org.opensearch.integration.storm
-#logger.OpenSearch-Storm.level=trace
\ No newline at end of file
+#logger.OpenSearch-Storm.level=trace
+
+
+
+logger.category.org.openearch.hadoop.mr=DEBUG
```

Comment:
I think this was just for debugging, can we remove it?

Reply:
Removed debugging.